src/main/java/io/jenkins/plugins/orka/helpers/Utils.java

```diff
@@ -13,4 +16,28 @@ public static String getErrorMessage(ResponseBase response) {
         return String.format("HTTP Code: %s, Error: %s", response.getHttpResponse().getCode(),
                 response.getErrorMessage());
     }
-}
\ No newline at end of file
+
+    public static int normalizeIdleTime(int userValue, int recommended) {
+        return userValue > 0 ? userValue : recommended;
+    }
+
+    public static FormValidation checkInputValue(String userInput, int recommended) {
+        try {
+            int idleMinutesValue = Integer.parseInt(userInput);
+
+            if (idleMinutesValue <= 0) {
+                return FormValidation.error("Idle timeout must be a positive number.");
+            }
+            
+            if (idleMinutesValue < recommended) {
```

Comment:
Let's remove the `not recommended` logic. I know it is what we discussed in the beginning, but now once I've tested it, it just looks strange.
If I want to set it to 10 we shouldn't warn you about it.

Reply:
Makes sense.